monthly aggregation

parameshwarbhat411 opened the pull request:
Added fact_commits_monthly.sql and fact_stars_monthly.sql models to the project

parameshwarbhat411 committed:
Added fact_commits_monthly.sql and fact_stars_monthly.sql models to the project

edsioufi commented on transform/models/marts/fact_commits_monthly.sql:
```diff
@@ -0,0 +1,12 @@
+  d.date_month,
+  dr.repo_name,
+  COUNT(*) as count_stars,
+  LAG(count_stars,1) OVER(PARTITION BY dr.repo_name ORDER BY d.date_month)as last_month_count_stars,
```
Naming: Should be `last_month_count_commits`

parameshwarbhat411 replied:
Gotchaa

edsioufi reviewed (changes requested):
Awesome stuff, just a minor naming mismatch that slipped through!

parameshwarbhat411 committed:
Refactored initial ingestion code to handle initial and incremental load. Made it more modular.
Added temp.conf for logging purpose
Added main class which triggers the whole flow
